samples/textDetection.js

```diff
@@ -185,74 +187,79 @@ function getTextFromFiles(index, inputFiles, callback) {
 }
 
 // Run the example
-function main(inputDir, callback) {
-  const index = new Index();
+function main(inputDir) {
+  return new Promise((resolve, reject) => {
```

Comment:
Please file an issue in the repo to come back and get rid of the use of `async` here :)

Reply:
You mean, to remove usage of "async.waterfall" and "async.parallel" Right?

Reply:
Issue filed [#249](https://github.com/googleapis/nodejs-vision/issues/249). I'll look into that in one or 2 days